app/templates/main.html

```diff
@@ -12,20 +12,13 @@
 
```

Comment:
Please also move the header and the footer back to `index.html`. Those should be visible on all pages.